Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+# Math Fundamentals
+
+# What you will learn 
+In this guide, you'll learn what is the fundamental mathematical knowledge you need to have in order
+to build insightful Data Science projects and build machine learning systems upon them.
+
+You'll also learn that not so much is needed to start to apply yourself to real-world data problems!
+
+## Prerequisites
+The previous Paradiso high-level guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Motivation](#Motivation)
+- [Courses](#Courses)
+  - [Calculus](#Calculus)
+  - [Linear Algebra and Matrix Algebra](#Linear-Algebra)
+  - [Optimization](#Optimization)
+- [Ask Questions](#Ask-Questions)
+- [Conclusions](#Conclusions)
+
+Let's dive right in!
+ 
+### Motivation
+
+Before anything else, watch [this](https://www.youtube.com/watch?v=VIbjHIGMjQM
+) and then [this](https://www.youtube.com/watch?v=8CX-Q0gtSp8) awesome Ted Talks :-)
+
+**The entire field of Data Science is based on mathematics and statistics.**
+
+This phrase shouldn't frighten you, although you may find out, that you probably have much of the knowledge you need to start experimenting!
+
+**In fact:** 
+
+If you've done a second-degree school (you know matrices and derivatives, for example), you probably just need to refresh your memory on the things you already know and go deeper into some things you may not have seen at school. 
+
+In general, if you have a scientific degree (engineering, computer science, physics, etc.), it will probably be just a review, because in this guide _you will learn how learn_ the fundamental theory behind Calculus, Linear Algebra, and Matrix Algebra courses are suggested within this guide.
```

Comment:
```suggestion
In general, if you have a scientific degree (engineering, computer science, physics, etc.), it will probably be just a review, because in this guide _you will learn how to learn_ the fundamental theory behind Calculus, Linear Algebra, and Matrix Algebra courses are suggested within this guide.
```